docs/modules/ROOT/pages/how-to/vshn-example/activate-global.adoc

```diff
@@ -46,3 +48,16 @@ Client Roles = appuio-cloud-realm
 Assigned Roles = view-users
 ----
 <1> The tabs are visible after saving the new user first.
+
+== Configure Cloudscale Metrics Collector
+
+. Login to Vault.
+. Login to control.cloudscale.ch.
+. Create a new API token within the `VSHN AppCat objects-lpg-2` project, give it a memorable name like `cloudscale-metrics-collector` and read-only access.
```

Comment:
Is this value hard-coded somewhere? `VSHN AppCat objects-lpg-2`

Reply:
WDYM with hardcoded? It's a changeable display name, manually set up by someone who has enough permissions on control.cloudscale.ch to create projects.
I wrote the instructions as if you're ever to recreate the global environment, that's the current project to use. When we launch the objectstorage on other clusters, we need to update these instructions probably

Reply:
As some parts are very specific we could maybe also mention the specific vault url + login url?

Reply:
added a snippet